content/departments/bizops/tools/amplitude.md

```diff
@@ -9,97 +9,36 @@
-
-If you're adding a new event property, please add it to the [data map](https://docs.google.com/spreadsheets/d/1wz958I67BKWWY0jKY3oXKhlrGZ9ucKmv0CM94K-5NVs/edit#gid=408201559).
-
-```
-JSON_EXTRACT(argument,
-  '$.code_search.results.results_count') AS results_count,
-JSON_EXTRACT(argument,
-  '$.code_search.results.alert') AS alert,
-CASE
-  WHEN name = 'ViewRepogroup:python' THEN 'python'
-  WHEN name = 'ViewRepogroup:golang' THEN 'golang'
-  WHEN name = 'ViewRepogroup:android' THEN 'android'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:kubernetes' THEN 'kubernetes'
-  WHEN name = 'ViewRepogroup:stanford' THEN 'stanford'
-  WHEN name = 'ViewRepogroup:cncf' THEN 'cncf'
-ELSE
-NULL
-END
-repogroup_name,
-```
-
-### Adding user properties
-
-User properties are the attributes of individual users. The `amplitude_user_characteristics` table defines all user properties, and we join in this table with every event that we pass to Amplitude. These user properties include every A/B test from `ab_test_users`. See more in the A/B testing doc (coming soon).
-
-If an additional user property is added to this table, it will only be applied to events and users going forward. User properties are sent to Amplitude along with events, so new properties will only be added when a user triggers an event that is sent to Amplitude. When adding a user property, please add it to the [data map](https://docs.google.com/spreadsheets/d/1wz958I67BKWWY0jKY3oXKhlrGZ9ucKmv0CM94K-5NVs/edit#gid=735397811)
+1. Events are stored in the [dotcom_events.events_usage](<[url](https://console.cloud.google.com/bigquery?project=telligentsourcegraph&_ga=2.7211002.912372869.1639779794-1385560724.1639779794&pli=1&ws=!1m5!1m4!4m3!1stelligentsourcegraph!2sdotcom_events!3sevents_usage)>)
+2. Incremental events are formatted to fit Amplitude's schema, enhanced with user properties (company name, company region, etc) in [this scheduled query](<[url](https://console.cloud.google.com/bigquery/scheduled-queries/locations/us/configs/6319611b-0000-203e-8729-94eb2c0924fc/runs?project=telligentsourcegraph)>), and a JSON is created in a [GCS bucket](<[url](https://console.cloud.google.com/storage/browser/event-usage-to-amplitude/data/2022/08;tab=objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&project=telligentsourcegraph&prefix=&forceOnObjectsSortingFiltering=false)>).
```

Comment:
I would change the link from `/data/2022/08` to just `/data/`
full link here: 
https://console.cloud.google.com/storage/browser/event-usage-to-amplitude/data?project=telligentsourcegraph&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false